src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,63 @@ title: Sign up for CodeStream
 description: "An overview of CodeStream's accounts and organizations and how to set those up for your team."
 ---
 
-To get the most out of New Relic CodeStream's collaboration tools, create an organization and then invite your team members to it. 
+To get the most out of New Relic CodeStream's collaboration tools, create an
+organization and then invite your team members to it. 
 
-If you haven't already, [sign up for a free New Relic account](https://newrelic.com/signup) to jump from your application's stack trace errors in New Relic One directly to the line of code responsible in your IDE.
+If you haven't already, [sign up for a free New Relic
+account](https://newrelic.com/signup) to jump from your application's stack
+trace errors in New Relic One directly to the line of code responsible in your
+IDE.
 
 ## Create an account [#account]
 
-If you already have the CodeStream extension [installed in your IDE](/docs/codestream/start-here/install-codestream), you can start the sign up process from the CodeStream pane.
+If you already have the CodeStream extension [installed in your
+IDE](/docs/codestream/start-here/install-codestream), you can start the sign up
+process from the CodeStream pane.
 
 ![Create an account](./images/CreateAnAccount6.png)
 
-There are two ways to sign up for CodeStream. You can sign up with a set of CodeStream credentials (such as, email address and password). Alternatively, you can sign up using your GitHub, GitLab, or Bitbucket (cloud versions only) account. 
+You can sign up using your email address and a password, or you can sign up using
+your GitHub, GitLab, or Bitbucket account (cloud versions only, although
+CodeStream does have integrations with the on-prem versions once you've
+signed up). If you're a New Relic customer, you can also sign up with a user api
+key from your New Relic One account.
 
-Signing up via your code host also connects your repositories to CodeStream. If the email address you're using with your code host isn't your work email, you should create CodeStream-specific credentials instead.
+If the email address you're using with your code host isn't your work email, you
+should sign up with your work email address instead.
```

Comment:
```suggestion
If you're not sure what email to use when you sign up, use your work email address.
```

Reply:
This copy needs to specifically refer to the code host auth path, because the user isn't deciding which email to use, they're deciding which auth method to use. And we need to discourage them from using their code host if they aren't using their work email with their code host.

Reply:
I think we can make more clear. All of what I know about how this works is based on this doc. I feel like I'm fumbling in the dark. 

How does someone make this choice? How do I know what my "code host auth path" is? :D How do I know whether I should use my work email or my code host?

In general, if I see an option to login to something via GitHub, I jump at the chance, because I'd usually rather not create a new login and password for some service I'm not sure about. But that's just me.

I find this all rather confusing.

I'd prefer something like:

"You have three different ways to create an account with CodeStream.
1. If you have a New Relic account, sign up with your user API key. (link to how to do that)
2. If you're joining your company's CodeStream organization, use your work email address.
3. If you use GitHub, GitLab, or Bitbucket and the email you use is the same as your work email, you can login through your code host provider."